src/pool.c

```diff
@@ -3463,7 +3463,11 @@ miner_on_submit(json_object *message, client_t *client)
     if (BN_cmp(hd, bd) >= 0)
     {
         /* Yay! Mined a block so submit to network */
-        log_info("+++ MINED A BLOCK +++");
+        log_info("+++ MINED A BLOCK +++ %.12s %"PRIu64" / %"PRIu64 " %"PRIu64,
+                 client->address,
+                 pool_stats.round_hashes,
```

Comment:
There's another small discrepancy with this line as `round_hashes` doesn't yet include the _current_ job's diff (the job that mined the block). Hence if you want accuracy, this should be `pool_stats.round_hashes + job->target`. If you're using this log line (and effort) for some kind of reward, you should probably fix it.